.travis.yml

```diff
@@ -8,6 +8,9 @@ go:
   - 1.9.1
 install:
   - make bootstrap
+before_script:
+  - make deps
```

Comment:
`make deps` required here ?

Reply:
I had to include it here, because cycle lint was failing because of missing deps.